front/src/constants.ts

```diff
@@ -5,12 +5,12 @@
 export const MAX_SAMPLERATE = 500_000;
 export const MIN_SAMPLERATE = 4000;
 export const MIN_DB = -140;
-export const DEFAULT_WINDOW_SIZE = 0.025;
-export const DEFAULT_HOP_SIZE = 0.5;
-export const DEFAULT_WINDOW = "hann";
```

Comment:
Happy to change the defaults, but it would be great to hear your thoughts on why 
these are better.